second.ino

```diff
@@ -60,15 +60,15 @@ float dt;
 float milliOld_mpu;
 float milliNew_mpu;
 float dt_mpu;
-int error = 0;
+int error = 0; // ?? Why there are so many global variables here while we can use them in functions only 
 float previousError = 0;
 float ErrorChange;
 float Slope;
 float Area = 0;
 int pid;
-int counter = 0;
-char arr[5] = {'f', 'f', 'r', 'r', 'r'};
-int d[5];
+int counter = 0;// ?? What is the usage of this counter
```

Comment:
This was used for a workaround as I recall, because the robot sometimes confuses a fork with a left/right turn.